swap_meet/clothing.py

```diff
@@ -0,0 +1,19 @@
+from .item import Item
+
+
+class Clothing(Item):
```

Comment:
👍 nicely done! the `super` method looks great! 

Also, as a note: I would put your doctstring right *above* the method or function you are summarizing. That way you don't break up the function